charts/seed-terraformer/charts/gcp-infra/values.yaml

```diff
@@ -21,3 +21,4 @@ networks:
   services: 100.64.0.0/13
   pods: 100.96.0.0/11
   worker: 10.250.0.0/19
+  internal: 10.250.112.0/22
```

Comment:
This should be commented out, otherwise this CIDR range will be used for every shoot cluster that does not specify an internal network range.